site/docs/v1/tech/apis/scim/_scim-enterpriseuser-response-body.adoc

```diff
@@ -0,0 +1,69 @@
+Because SCIM schemas are completely customizable, The SCIM response returned is generated from the result of SCIM User object populated by your outgoing response lambda.
+
+The following is an example of a typical SCIM User response body that your SCIM client might expect. However, your outgoing response lambda could generate a completely different SCIM schema is desired.
```

Comment:
is it a "typical SCIM User response body that your SCIM client might expect" or one that our default lambda implementation returns, or both? I think the latter is more important to mention.